tests/test_imports.py

```diff
@@ -11,61 +20,148 @@
+    exp_url = 'https://data.microbiomedata.org/data/nmdc:omprc-11-importT/52834.4.466476.GATCGAGT-GATCGAGT.fastq.gz'
+    exp_name = '52834.4.466476.GATCGAGT-GATCGAGT.fastq.gz'
+    exp_description = 'Metagenome Raw Reads for nmdc:omprc-11-importT'
+
+    db, update = gold_mapper.map_sequencing_data()
+    # Database assertions
+    assert db
+    assert isinstance(db, Database)
+    assert db.data_object_set
+    data_objects = db.data_object_set
+    assert len(data_objects) == exp_num_data_objects
+    # Data object assertions
+    dobj = data_objects[0]
+    assert isinstance(dobj, DataObject)
+    assert str(dobj.data_object_type) == exp_dobj_type
+    assert dobj.id == exp_dobj_id
+    assert dobj.name == exp_name
+    assert dobj.description == exp_description
+    assert dobj.url == exp_url
+    assert dobj.file_size_bytes
+    assert dobj.md5_checksum
+
+    # Update assertions
+    assert update
+    assert update == exp_update
+
+
+def test_gold_mapper_map_data_unique(gold_mapper):
     """
-    This test counts the number of files from gold_mapper where the data object creation should be 1:1.
+    Test that the gold mapper creates data objects for the data files other
+    than the sequencing data
     """
-    gold_mapper.unique_object_mapper()
-    assert len(gold_mapper.nmdc_db.data_object_set) == 2
-    assert len(gold_mapper.objects) == 2
+    initial_num_data_objects = 1
+    db, update = gold_mapper.map_sequencing_data()
+    # sanity check
+    assert len(db.data_object_set) == initial_num_data_objects
+        # two unique data files from the gold import files fixture get added to the database
+    exp_data_object_types = [
+        "Clusters of Orthologous Groups (COG) Annotation GFF",
+        "Pfam Annotation GFF",
+        "Metagenome Raw Reads",
+        "Annotation Amino Acid FASTA",
+        "Filtered Sequencing Reads",
+        "Assembly Contigs",
+    ]
+
+    exp_nucleotide_sequencing_id = "nmdc:omprc-11-importT"  # From the gold mapper fixture
+
+    db, do_map = gold_mapper.map_data(db)
+    assert db
+    assert len(db.data_object_set) == len(exp_data_object_types)
+    data_objects = db.data_object_set
+    for dobj in data_objects:
+        assert str(dobj.data_object_type) in exp_data_object_types
+        assert isinstance(dobj, DataObject)
+        assert dobj.url
+        assert exp_nucleotide_sequencing_id in dobj.url
+        assert exp_nucleotide_sequencing_id in dobj.description
 
 
-def test_multiple_object_mapper(gold_mapper):
+
+def test_gold_mapper_map_data_multiple(gold_mapper):
     """
-    This test counts the number of files from gold_mapper where the data object creation should be many:1.  JGI stores each binning file 
-    individually whereas NMDC combines all the records into a single tar.gz file.
+    Test that the gold mapper creates data objects for the data files other
+    than the sequencing data
```

Comment:
clarify description that this is a test for when there are 2 or more files being repackaged as 1 file with 1 corresponding data object.